apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -19,21 +22,36 @@ import moduleStyles from './chatWorkspace.module.scss';
 interface ChatWorkspaceProps {
   onClear: () => void;
 }
+interface Students {
+  [index: number]: {
+    id: number;
+    name: string;
+  };
+}
+
+const WORKSPACE_VIEW_MODE = {
```

Comment:
We should use an enum / union type for this so we can type the state value

Also - this feels a little overloaded to me since it determines both which tab is being shown as well as whether to show the tabs or not? I think it would be good to decouple those two sets of values - maybe if we can compute whether to show the tabs or not based off of `viewAsUserId` as mentioned below, these values can just determine which tab is being shown?

Reply:
Good points! Decoupled, renamed, and cleaned up. Thanks!